ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs

```diff
@@ -84,6 +84,11 @@ public static class ViewProps
         public const string BorderBottomLeftRadius = "borderBottomLeftRadius";
         public const string BorderBottomRightRadius = "borderBottomRightRadius";
 
+        public const string ShadowColor = "shadowColor";
+        public const string ShadowOffset = "shadowOffset";
```

Comment:
Can we just inline the strings?  I try to keep tis file in sync with [https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewProps.java](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewProps.java).  I should have put a comment along those lines in here.